src/functions/pollJournals/framework/repo/mysql/row-mappers/personal-commitment-row-mapper.ts

```diff
@@ -0,0 +1,26 @@
+import { ExaminerPersonalCommitment } from '../../../../domain/examiner-personal-commitment';
+
+interface PersonalCommitmentRow {
+  individual_id: number;
+  commitment_id: number;
+  start_date_time: string;
+  end_date_time: string;
+  non_test_activity_code: string;
+  reason_desc: string;
+}
+
+// TODO: Split out start/end date/time
```

Comment:
Would this be something that could be implemented now?

Reply:
Sorted this, thanks for pointing out.